src/apps/favorites-list/FavoritesList.tsx

```diff
@@ -44,8 +44,8 @@ const FavoritesList: React.FC<FavoritesListProps> = ({ pageSize }) => {
 
   const skeletonList = (
     <>
-      <div className="ssc">
-        <div className="ssc-line w-10 my-32">&nbsp;</div>
+      <div className="ssc content-list">
+        <div className="ssc-line w-10 mt-32">&nbsp;</div>
```

Comment:
This is tailwind classes, or? 

Just currious 😄

Reply:
hihi, nopes - we're using a skeleton screen package that has some classes 😉 like ssc, ssc-line. and w-10